tools/fixFormat.js

```diff
@@ -83,6 +83,17 @@ async.each(packages, function(item, callback) {
         pkg.autoupdate.basePath = basePath;
     }
   }
+  if (pkg.keywords && (pkg.keywords != undefined)) {
+    var kw = pkg.keywords;
+    for (var i = 0; i < kw.length; i++) {
+      for (var j = i+1; j < kw.length; j++) {
+        if (kw[i] === kw[j]) {
+            kw.splice(j, 1);
+            j--;
+        }
```

Comment:
Please try to write this part more elegant like this example:
https://github.com/jsifalda/filter-duplicates/blob/master/index.js
```js
function arrayUnique (array) {
  return array.filter(function (item, position) {
    return array.indexOf(item) === position
  })
}
```